Add Welsch instructions for ED2 runs

KristinaRiemer opened the pull request.

KristinaRiemer commented:
Only have ED2 working currently in the web interface, still working on workflow.R.

KristinaRiemer commented:
@jeanetteclark I'm assigning you as the reviewer for the PR because you're interested in learning more about running models? If so, the entire .Rmd should knit locally and I also want to make sure the instructions in the second section (*ED2 Run Using University of Arizona's Web Interface of PEcAn*) are clear and work.

jeanetteclark commented:
Sweet! I can probably get to this some time this week. By locally you mean I should be able to clone this repo to my local machine and run? Or I run this on welsch?

KristinaRiemer commented:
Oof, yeah, I wasn't clear about that. I meant that you should be able to clone the repo locally and then knit the .Rmd successfully. But in order to actually reproduce the figure at the end of the section, you'll have to use Welsch according to the instructions. Does that make sense?

jeanetteclark commented on ED2/ed2_in_pecan.Rmd:
```diff
@@ -155,3 +155,324 @@ library(ggplot2)
+
+# 2. ED2 Run Using University of Arizona's Web Interface of PEcAn
+
+Access the web interface by navigating to [welsch.cyverse.org:8000/](welsch.cyverse.org:8000/) in the browser and clicking on the link that is the first bullet in the **Applications** section, which states *PEcAn web interface, from here you can run different models at different sites.* 
```
edited for clarity

KristinaRiemer commented:
@jeanetteclark thanks for this feedback! I think I've addressed all of your comments, including adding the missing instructions for after editing the config file. Hopefully that is helpful? 

In terms of your general comment about why certain parameters, etc. were chosen, I added a sentence at the beginning to clarify a bit. These tutorials aren't really supposed to be general purpose for anyone using ED2 in PEcAn, they're specifically for me to document what I'm doing for the DARPA project and for other people on the project to be able to replicate what I've done. There's currently nothing in there specific to the project, so all the settings in the document are just randomly chosen to get ED2 to run. I know that's not very satisfying, but once I start including parameters and sites for the project, I'll definitely include justifications for those things!

jeanetteclark commented on ED2/ed2_in_pecan.Rmd:
```diff
@@ -6,9 +6,11 @@ output: github_document
 ---
 
-## Run model
+The specific settings chosen for sections 1-3 of this vignette, including plant functional typea (PFT), sites, and date ranges were chosen as examples to illustrate how to use the PEcAn infrastructure to run the ED2 model. 
```
```suggestion
The specific settings chosen for sections 1-3 of this vignette, including plant functional type (PFT), sites, and date ranges were chosen as examples to illustrate how to use the PEcAn infrastructure to run the ED2 model. 
```

KristinaRiemer replied:
Good catch!
